drydock/templates/drydock/k8s/patches/pdb.yml

```diff
@@ -0,0 +1,74 @@
+{%- if MFE_HOST is defined %}
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: mfe-pdb
+spec:
+  minAvailable: 30%
```

Comment:
This value should be configurable.

Also I think we should be able to turn off the PDBs individually. We could probably take a similar approach to open-craft's Grove where certain values of `minAvailable` disable the pdb (i.e. `-1`).